client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { localize } from 'i18n-calypso';
+import {
+	getTwoFactorUserId,
+	getTwoFactorAuthNonce,
+	isTwoFactorAuthTypeSupported,
+} from 'state/login/selectors';
+import { sendSmsCode } from 'state/login/actions';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import { login } from 'lib/paths';
+
+class TwoFactorActions extends Component {
+	static propTypes = {
+		errorNotice: PropTypes.func.isRequired,
+		successNotice: PropTypes.func.isRequired,
+		isSmsSupported: PropTypes.bool.isRequired,
```

Comment:
`isSmsSupported` can be null ([source](https://github.com/Automattic/wp-calypso/blob/master/client/state/login/selectors.js#L121)) so it shouldn't be a required prop.